.github/workflows/tests.yml

```
@@ -80,11 +80,20 @@ jobs:
             registry: ghcr.io
             username: ${{ github.actor }}
             password: ${{ secrets.GITHUB_TOKEN }}
+        - name: Sanitize Docker tag
+          id: vars
+          run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:6:43: Double quote to prevent globbing and word splitting

(shellcheck)